onnxruntime/core/providers/shared_library/provider_interfaces.h

```diff
@@ -25,7 +25,8 @@ using DataType = const std::string*;
 
 namespace onnxruntime {
 
-// onnx Protobuf types
+// onnx Protobuf types (all of these are actually just Provider_<type> -> ONNX_NAMESPACE::<type>)
+#ifndef PROVIDER_BRIDGE_ORT
```

Comment:
>PROVIDER_BRIDGE_ORT [](start = 8, length = 19)

Where is this defined?

Reply:
It's an existing macro defined by providers_bridge_ort.cc to have it not see everything in provider_interfaces.h (and with this change it makes it see even less).